helpers/z_schema.js

```diff
@@ -105,6 +106,13 @@ z_schema.registerFormat('ip', function (str) {
 	return ip.isV4Format(str);
 });
 
+/* Since an IP is not considered to be a hostname 
```

Comment:
We can remove this comment and change the header of the file.

Reply:
@diego-G Done.